Pinta/Dialogs/DocumentPropertiesDialog.cs

```diff
@@ -0,0 +1,202 @@
+// 
+// DocumentPropertiesDialog.cs
+//
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Gtk;
+using Mono.Unix;
+using Pinta.Core;
+
+namespace Pinta
+{
+    public class DocumentPropertiesDialog : Dialog
+    {
+        private DocumentProperties initial_properties;
+
+        public string author;
+        public string title;
+        public string subject;
+        public string keywords;
+        public string comments;
+
+        private Entry authorEntry;
```

Comment:
Nitpick: Pinta's source code is fairly inconsistent about how private members are named, but it should at least be consistent within a class - `lower_case` is what I prefer